android-sdk/src/main/java/org/sagebionetworks/bridge/android/di/BridgeApplicationModule.java

```diff
@@ -0,0 +1,15 @@
+package org.sagebionetworks.bridge.android.di;
+
+import android.app.Application;
+import android.content.Context;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module(includes = {BridgeServiceModule.class, S3Module.class})

```

Comment:
I'd probably be much more pro-dagger if it was all in Kotlin and you had a single file with all of these provider and injector classes defined within it (Java doesn't allow that). Instead, I find myself wasting countless hours because there are so many of these little provider files which are nearly impossible to track down what is happening unless you wrote them yourself or are an expert-level dagger dev.  It's all very obfuscated to me.

Reply:
I agree 100%. When fixing the dependency issue it was obvious the modules and scoping related to SR were not properly organized